yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -868,6 +872,46 @@ func generateAssessmentReport() (err error) {
 	return nil
 }
 
+func addAssessmentIssuesForRedundantIndexes() error {
+	if source.DBType != POSTGRESQL {
+		return nil
+	}
+	query := fmt.Sprintf(`SELECT redundant_schema_name,redundant_table_name,redundant_index_name,
+	existing_schema_name,existing_table_name,existing_index_name,
+	redundant_ddl,existing_ddl from %s`,
+		migassessment.REDUNDANT_INDEXES)
+	rows, err := assessmentDB.Query(query)
+	if err != nil {
+		return fmt.Errorf("error querying-%s on assessmentDB for redundant indexes: %w", query, err)
+	}
+	defer func() {
+		closeErr := rows.Close()
+		if closeErr != nil {
+			log.Warnf("error closing rows while fetching redundant indexes %v", err)
+		}
+	}()
+
+	var redundantIndexesInfo []utils.RedundantIndexesInfo
+	for rows.Next() {
+		var redundantIndex utils.RedundantIndexesInfo
+		err := rows.Scan(&redundantIndex.RedundantSchemaName, &redundantIndex.RedundantTableName, &redundantIndex.RedundantIndexName,
+			&redundantIndex.ExistingSchemaName, &redundantIndex.ExistingTableName, &redundantIndex.ExistingIndexName,
+			&redundantIndex.RedundantIndexDDL, &redundantIndex.ExistingIndexDDL)
+		if err != nil {
+			return fmt.Errorf("error scanning rows for redundant indexes: %w", err)
+		}
+
+		redundantIndexesInfo = append(redundantIndexesInfo, redundantIndex)
+	}
+	redundantIssues := parserIssueDetector.GetRedundantIndexIssues(redundantIndexesInfo)
+	for _, redundantIssue := range redundantIssues {
+		convertedAnalyzeIssue := convertIssueInstanceToAnalyzeIssue(redundantIssue, "", false, false)
```

Comment:
nit: would have preferred directly converting redundantIssue to an AssessmentIssue, but ignore suggestion if this makes it easy.

Reply:
I think this is okay, we already had the handling for some fields like (category etc.. ) in the existing functions, so better to use them.